app/Services/LdapAd.php

```diff
@@ -542,13 +543,16 @@ public function testUserImportSync(): Collection
      */
     public function getLdapUsers(): Paginator
     {
+        \Log::debug("IO don't thinkk it fired yet");
         $search = $this->ldap->search()->users()->in($this->getBaseDn()); //this looks wrong; we should instead have a passable parameter that does this, and use this as a 'sane' default, yeah?
 
         $filter = $this->getFilter();
         if (!is_null($filter)) {
             $search = $search->rawFilter($filter);
         }
-
+        \Log::debug("I thinkn *THIS* is where it actually fires, yeah?");
```

Comment:
Typo - also do we still need this debugging? (It's okay if we do - most folks wouldn't be running this in debug mode)